docs/examples/index.md

```diff
@@ -0,0 +1,9 @@
+---
+description: |
+  In this chapter of the Fresh documentation, you can find examples of features that you may like in your Fresh project.
+---
+
+In this chapter of the Fresh documentation, you can find examples of features
+that you may like in your Fresh project. If there's a specific example you'd
+like to see here, please let us know in
+[a GitHub issue](https://github.com/denoland/fresh/issues/new).
```

Comment:
Discussions instead maybe?

Reply:
Yeah. Actually, discussions would be better because the example idea would likely achieve greater consensus before becoming an actionable issue.